language/diem-framework/modules/PaymentScripts.move

```diff
@@ -71,7 +71,7 @@ module PaymentScripts {
         );
         DiemAccount::restore_withdraw_capability(payer_withdrawal_cap);
     }
-    spec fun peer_to_peer_with_metadata {
+    spec fun p2p {
```

Comment:
I think it is better to keep name consistent, using "peer_to_peer_with_metadata" instead of p2p

Reply:
What would you think of `peer_to_peer` instead? Or is your main concern consistency with the old transaction script name?

Reply:
I think consistent with the old transaction script name is better.
We have example code for payment transactions, it will be hard to find out all of them and update. People can be much easier to map to new script function by the name.